### ISSUE TITLE
Built Executable Flagged as Virus and Instantly Deleted on Windows

### ISSUE THREAD

**Issue by patrickkdev** [wailsapp/wails#3026]
### Description

I successfully developed a simple application using the framework, and everything worked flawlessly during the development process. However, when I deployed the application to production and made it available for download on my website, I encountered a critical issue. Upon attempting to download the application on different computers, including those with only Windows Defender, it consistently triggered virus warnings.

Both Chrome and Edge raised alarms, and despite granting permission to proceed with the download, Windows Defender also issued warnings about a potential threat. Additionally, I attempted to create an installer using NSIS, but after successful installation, Windows promptly removes the executable file. I have searched the issue tracker, discussion forums, and the Troubleshooting Guide but have not found any pertinent information to resolve this problem.

### To Reproduce

**Steps to Reproduce:**

1. Create a new Wails app using the following command:
   ```
   wails init -n myproject -t react-ts
   ```

2. Navigate to the project directory:
   ```
   cd myproject
   ```

3. Install project dependencies using Yarn:
   ```
   yarn
   ```

4. Return to the project's root directory:
   ```
   cd ..
   ```

5. Build the application:
   ```
   wails build
   ```

6. Locate the executable within the `build/bin` directory.

7. Run a virus checker, such as VirusTotal, against the executable.

   - **Note:** The executable will be detected as a virus by some of the anti viruses
   
https://www.virustotal.com/gui/file/cec9992d2e908e81a2849c983e0b06d722b445da83146509b7de1bd020a33796?nocache=1
https://mega.nz/file/htBD0C6Q#wQDvwtkD3a_yI5_QR3q7SneUl2pvUImMmilShrsDEYE

### Expected behaviour

After following the provided steps, the executable generated by the Wails application should be free from any virus detection when scanned by a reputable antivirus software, such as VirusTotal. The application should be deemed safe and not trigger any virus alerts during the scan, ensuring that users can download and run the application without any security concerns.

### Attempted Fixes

I attempted to resolve the problem by creating a completely new Wails app. However, even with a freshly created app, the issue persists as the new app is also flagged as a virus during virus scans.

### System Details

```shell
DEB | Using go webview2loader

                                
          Wails Doctor          
                                


# Wails
Version | v2.6.0

# System
┌────────────────────────────────────┐
| OS           | Windows 10 Pro      |
| Version      | 2009 (Build: 22621) |
| ID           | 22H2                |
| Go Version   | go1.21.1            |
| Platform     | windows             |
| Architecture | amd64               |
└────────────────────────────────────┘

# Dependencies
┌───────────────────────────────────────────────────────┐
| Dependency | Package Name | Status    | Version       |
| WebView2   | N/A          | Installed | 118.0.2088.76 |
| Nodejs     | N/A          | Installed | 19.4.0        |
| npm        | N/A          | Installed | 9.2.0         |
| *upx       | N/A          | Available |               |
| *nsis      | N/A          | Installed | v3.09         |
└─────────────── * - Optional Dependency ───────────────┘

# Diagnosis
Optional package(s) installation details:
  - upx : Available at https://upx.github.io/

 SUCCESS  Your system is ready for Wails development!
```

**Comment by stffabi** [comment]
Thanks for using Wails and reporting this.

Unfortunately there's nothing we can do about this from our side, it is a Go ecosystem problem. Please also see #2965

For Windows you can get around that by purchasing a valid [CodeSigning](https://wails.io/docs/guides/signing/) certificate and sign your application with it.

**Comment by mklnz** [comment]
I understand there is nothing Wails can do about this issue but I'm having a lot of problems with Windows Defender falsely detecting my app and is holding up deployment.

I've submitted the app to Microsoft Security Intelligence many many times and it's been detecting on and off despite their reply being it is "fixed".

I guess my post isn't to complain but to ask what else we can do to reach out to Microsoft (beyond level 1 people reviewing submissions) and rectify this problem long term for Go users and Wails users alike.

This is essentially a show stopper for any of us wanting to use Wails to deploy desktop applications on Windows. I'm almost tempted to rewrite the entire app in .NET simply because I cannot count on Microsoft ever fixing this through false positive submissions.

**Comment by leaanthony** [comment]
That's so frustrating! There must be something about the way your app appears to Windows Defender. What channel did you use to talk to Microsoft? I can try talking to them but they're likely just going to say "buy a signing certificate".

**Comment by mklnz** [comment]
This is extremely frustrating, different to other engineering issues where you know how to fix it and just takes time. This is more of a human problem and actually getting a huge corporation to listen to you.

Perhaps there is something on how my app appears, but the thing is it was never a problem starting from Jan 2024 after I refactored some code and released an update. Now sometimes even a Hello World go application gets flagged upon compilation.

The thing is I have an EV code signing certificate and even that doesn't help. All the binaries are signed using a private key stored on a hardware token. Another separate unrelated issue is SmartScreen detecting it as unknown publisher even though I have an EV code signing cert (This is a whole other issue that is also extremely frustrating, more reading at: https://techcommunity.microsoft.com/t5/security-compliance-and-identity/codesigning-with-ecc-certificate-rather-than-rsa-works-with/m-p/4026498).

I've been using the official channel to submit false positives at https://www.microsoft.com/en-us/wdsi/filesubmission
I just submitted a support ticket via https://partner.microsoft.com as well, we'll see what they say.
